types/d3-indirections/d3-indirections-tests.ts

```diff
@@ -0,0 +1,62 @@
+/// <reference types="./index" />
```

Comment:
THis should be `<reference types="d3-indirections" />`, same as a user of the package would write.

Reply:
Thanks, I have updated it.